scripts/us_census/pep/us_pep_sex/process.py

```diff
@@ -0,0 +1,908 @@
+    df['geo_ID'] = df['geo_ID'] + '-' + df['Year'].astype(str)
+    df = df.drop(columns=['Year'])
+    df = df.groupby(['geo_ID','Race/Sex Indicator']).sum().transpose()\
+        .stack(0).reset_index()
+    df[['geo_ID', 'Year']] = df['geo_ID'].str.split('-', expand=True)
+    df = df.drop(columns=['level_0'])
+    df['Measurement_Method'] = 'CensusPEPSurvey'
+    return df
+
+
+def _county_1980_1990(file_path: str):
+    """
+    Process and cleans the file for county 1980 to 1990.
+
+    Args:
+        file_path (str) : input file path.
+
+    Returns:
+        df (pd.DataFrame) : cleaned dataframe.
+    """
+    df = pd.read_csv(file_path, skiprows=5)
+    df['Total'] = df[_COLUMNS_TO_SUM].sum(axis=1)
+    df = df.drop(columns=_COLUMNS_TO_SUM)
+    df = df.replace({
+        'Race/Sex Indicator': {
+            'White male': 'Count_Person_Male',
+            'Black male': 'Count_Person_Male',
+            'Other races male': 'Count_Person_Male',
+            'White female': 'Count_Person_Female',
+            'Black female': 'Count_Person_Female',
+            'Other races female': 'Count_Person_Female'
+        }
+    })
+    df = df.rename(columns={
+        'Year of Estimate': 'Year',
+        'FIPS State and County Codes': 'geo_ID'
+    })
+    df['geo_ID'] = 'geoId/' + (df['geo_ID'].map(str)).str.zfill(5)
+    df['geo_ID'] = df['geo_ID'] + '-' + df['Year'].astype(str)
+    df = df.drop(columns=['Year'])
+    df = df.groupby(['geo_ID','Race/Sex Indicator']).sum().transpose()\
+        .stack(0).reset_index()
+    df[['geo_ID', 'Year']] = df['geo_ID'].str.split('-', expand=True)
+    df = df.drop(columns=['level_0'])
+    df['Measurement_Method'] = 'CensusPEPSurvey'
+    return df
+
+
+def _county_1990_2000(file_path: str):
+    """
+    Process and cleans the file for county 1990 to 2000 and also aggregate
+    county values to get state 1990 to 2000.
+
+    Args:
+        file_path (str) : input file path.
+
+    Returns:
+        df (pd.DataFrame) : cleaned dataframe.
+    """
+    column_names = ['Year', 'geo_ID', 'Age', 'Race-Sex', 'Ethnic', 'Value']
+    df = pd.read_table(file_path, delim_whitespace=True, header=None)
+    df.columns = column_names
+    df['Year'] = '19' + df['Year'].astype(str)
+    df['geo_ID'] = 'geoId/' + (df['geo_ID'].map(str)).str.zfill(5)
+    df = df.drop(columns=['Age', 'Ethnic'])
+    df = df.replace({
+        'Race-Sex': {
+            1: 'Count_Person_Male',
+            2: 'Count_Person_Female',
+            3: 'Count_Person_Male',
+            4: 'Count_Person_Female',
+            5: 'Count_Person_Male',
+            6: 'Count_Person_Female',
+            7: 'Count_Person_Male',
+            8: 'Count_Person_Female'
+        }
+    })
+    df['geo_ID'] = df['geo_ID'] + '-' + df['Year'].astype(str)
+    df = df.drop(columns=['Year'])
+    df = df.groupby(['geo_ID','Race-Sex']).sum().transpose()\
+        .stack(0).reset_index()
+    df[['geo_ID', 'Year']] = df['geo_ID'].str.split('-', expand=True)
```

Comment:
It looks like we are concatenating geoId and Year; and later we are splitting it. Can you please check if this was required.

Reply:
We did this so that we while doing sum no extra rows are added and we get all the required columns in the output dataframe.